sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/RedactionFunctionsSuite.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.expressions
+
+import org.apache.spark.{SparkFunSuite, SparkRuntimeException}
+import org.apache.spark.sql.AnalysisException
+import org.apache.spark.sql.catalyst.analysis.TypeCheckResult
+
+class RedactionFunctionsSuite extends SparkFunSuite with ExpressionEvalHelper {
```

Comment:
Am I missing anything, `MASK_CCN_INVALID_FORMAT_ERROR` is not tested yet?

Reply:
It's covered in the `QueryCompilationErrors.maskCcnInvalidFormatError` method:

![image](https://user-images.githubusercontent.com/99207096/193899804-8e3e2487-ec4a-4464-a4e1-bc4eb722f554.png)

Reply:
oops yes thanks!